pkgs/development/tools/build-managers/buildbot8/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv,
```

Comment:
A common convention I've seen is for people to have folders for the project name, and then version-specific files inside it. So you might have buildbot/8.nix, buildbot/9.nix, and a default.nix that lets you access both of them.

It really depends on how long you think this will stick around though. In general we have a policy in nixpkgs that we try to avoid maintaining simultaneous multiple versions unless something else needs them, so if we're on 0.9 rc1, it seems like 0.8's days are numbered. Or if they're fundamentally different (in a python2 vs python3 way) then perhaps they'll both survive.